### ISSUE TITLE
Training Data Horizon

### ISSUE THREAD

**Issue by pi-menta1058** [shiyu-coder/Kronos#14]
Hello,

I have been reading your paper on Kronos and have started testing some of the trained models for forecasting. Congratulations on the excellent work, Kronos is a truly impressive, clever and innovative approach. I do have a couple of questions regarding the training dataset.

From what I understood, most of the data begins around 2010, but could you clarify until which year the training data extends? In addition, do you have specific numbers or details regarding the coverage of the Brazilian stock market within the dataset? 

Thank you in advance for your clarification.

**Comment by shiyu-coder** [comment]
We are glad that our work has been helpful and inspiring to you.

For detailed information about the entire dataset, you can refer to Table 13 in the appendix of the paper.

The cut-off date for the training set is February 2024 (inclusive). The validation set spans from March to May 2024 (inclusive), and the out-of-sample period begins in June 2024.

Our dataset covers 2058 stocks from the Brazil Stock Exchange. I am not sure what proportion of the entire Brazilian stock market this represents, as this is all the data we are able to obtain.

Thank you.
